### DISCUSSION TITLE
Ubuntu install suggestion is suspect

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#9133] [Troubleshooting] [answered]
https://github.com/helix-editor/helix/blob/585402d9ffa1d271e7c72fbb1c90d7e9d2d51482/book/src/install.md?plain=1#L61

I am on Pop-OS which is a derivative of Ubuntu. There is no `add-apt-repository` command. Was this a quick copy-paste job?

To add something positive, it looks like the way to do this is `apt edit-sources`. But in that case the format of the repository will be needed. Here's the spec I currently have for `swi-prolog`:

    deb https://ppa.launchpadcontent.net/swi-prolog/stable/ubuntu/ jammy main

**Comment by the-mikedavis** [comment, marked as answer]
`add-apt-repository` is https://manpages.ubuntu.com/manpages/trusty/man1/add-apt-repository.1.html. Looks like it comes from the `software-properties-common` package

**Reply by ghost** [reply]
I see, that's part of the `packagekit` stuff that I removed (because I dislike opaque GUI solutions to sysadmin tasks). So, I'll leave it up to you -- the suggestion is not invalid, but I still think that providing a full `sources.list` line would be better.

-- 
Ian
